frontend/src/scenes/notebooks/NotebookSelectButton/notebookSelectButtonLogic.ts

```diff
@@ -52,7 +52,7 @@ export const notebookSelectButtonLogic = kea<notebookSelectButtonLogicType>([
             [] as NotebookListItemType[],
             {
                 loadAllNotebooks: async (_, breakpoint) => {
-                    breakpoint(100)
```

Comment:
`breakpoint` calls with a delay argument but no `await` take no effect. So this didn't do much, except sometimes throw an uncaught in-promise error, which is just bad. So a heads-up that behavior may _slightly_ change here.
(I have a branch adding the `@typescript-eslint/no-floating-promises` rule to prevent this in the future, but will put up as a PR on Monday.)

Reply:
TIL thanks Michael